packages/logger/src/writers/null.ts

```diff
@@ -1,3 +1,3 @@
 import {LogWriter} from '@flumer/core';
 
-export const nullWriter: LogWriter = async (loggable, formatter) => {};
+export const nullWriter: LogWriter = async (_loggable, _formatter) => {};
```

Comment:
The `_` prefix for method parameter names is a convention used to notify that the parameter is provided on invocation but not used within the method implementation